ios/Podfile

```diff
@@ -1,7 +1,7 @@
 require_relative '../node_modules/react-native/scripts/react_native_pods'
 require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
 
-platform :ios, '12.4'
+platform :ios, '13.0'
```

Comment:
Same for iOS. We'll need thorough testing and to make sure no native SDK/function is deprecated with this OS version.

Reply:
This one is a _minimum_ version change so we shouldn't need to do anything other than notify Applause to stop testing < 13.0 devices